### PR TITLE
TC_04.01.012 |Freestyle project > Verify that user is able to disable the project

### PR DESCRIPTION
**Pull request by SafonovaOlya**
Added verification that user is able to disable the project

### CODE REVIEW

File: cypress/e2e/freestyleProjectGeneralSection.cy.js [hunk 1]
```diff
@@ -4,14 +4,27 @@ import freestyleProject from "../fixtures/freestyleProject.json"
...
+
+    it('TC_04.01.011 | Freestyle project > Verify that General section is displayed on Configuration page', function(){
+
         cy.get('[data-section-id="general"]').should('have.class', 'task-link--active')
         cy.get('#general').should('be.visible');
     })
+
+    it('TC_04.01.012 | Freestyle project > Verify that user is able to disable the project', function(){
+
+        cy.get('.jenkins-toggle-switch__label__checked-title').should('have.text', 'Enabled');
```
Comment (S-Lanchik) [inline cypress/e2e/freestyleProjectGeneralSection.cy.js:24]: put 'Enabled' to fixtures

File: cypress/e2e/freestyleProjectGeneralSection.cy.js [hunk 2]
```diff
@@ -4,14 +4,27 @@ import freestyleProject from "../fixtures/freestyleProject.json"
...
+        cy.get('#toggle-switch-enable-disable-project').click();
+
+        cy.get('.jenkins-toggle-switch__label__unchecked-title').should('have.text', 'Disabled');
```
Comment (S-Lanchik) [inline cypress/e2e/freestyleProjectGeneralSection.cy.js:27]: put 'Disabled' to fixtures

File: cypress/e2e/freestyleProjectGeneralSection.cy.js (hunk 1 above)
Comment (SafonovaOlya) [reply cypress/e2e/freestyleProjectGeneralSection.cy.js:24]: @S-Lanchik could you please explain why I should put "Enabled" and "Disabled" for this test case?

Comment (S-Lanchik) [reply cypress/e2e/freestyleProjectGeneralSection.cy.js:24]: If the text changes, you will need just update the text in fixtures. In that case, you don't need to find all test with this data and change text there.

Comment (SafonovaOlya) [reply cypress/e2e/freestyleProjectGeneralSection.cy.js:24]: got it. Thank you!

**Review by S-Lanchik — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Pull request merged**